RefugeeLand.Core.Api/Migrations/20220520211059_AddShelterModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace RefugeeLand.Core.Api.Migrations
+{
+    public partial class AddShelterModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
```

Comment:
I don't believe that this is going to create any tables. Can somebody explain how our shelter is being migrated. Maybe we forgot to put `DbSet<Shelter>` in the `StorageBroker.Shelters.cs` because I didn't see the changes... cc: @rcruzfreelance  @Spectralgo